CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 0.13.0
+
+- Add `thiserror` to dependencies.
+- Modify `Error` to be implemented using the `thiserror::Error` derive macro.
```

Comment:
Thanks for adding ChangeLog entries. I'd remove these first two as they are rather implementation details.